packages/walletconnect/src/index.js

```diff
@@ -167,7 +167,18 @@ export default class WalletConnect extends Connector {
     const result = await this._getEncryptedData(`/session/${this.sessionId}`)
 
     if (result) {
-      if (result.data.approved) {
+      if (result.data && !result.data.approved) {
```

Comment:
This part is just legacy to be able to use WallEth before upgrade. 
It's not important.